src/proto/FunctionRpc.proto

```diff
@@ -180,6 +184,16 @@ message WorkerStatusRequest{
 message WorkerStatusResponse {
 }
 
+message FunctionEnvironmentLoadRequest {
+  // Environment variables from the current process
+  map<string, string> environment_variables = 1;
```

Comment:
Can an environment variable's value ever be null or empty? languages like javascript will buffer overflow if so :|

Reply:
we can control this in host.